packages/d3fc-chart/src/cartesianBase.js

```diff
@@ -24,21 +24,13 @@ export default (setPlotArea, defaultPlotArea) =>
                     .style('height', '1em')
                     .style('line-height', '1em');
 
-                const yOrientValue = cartesian.yOrient()(data);
-
-                container.enter()
-                    .append('div')
-                    .attr('class', 'y-label-container')
-                    .style('grid-column', yOrientValue === 'left' ? 1 : 5)
-                    .style('-ms-grid-column', yOrientValue === 'left' ? 1 : 5)
-                    .style('grid-row', 3)
-                    .style('-ms-grid-row', 3)
-                    .style('width', '1em')
+                container
+                    .select('div.y-label-container')
                     .style('display', 'flex')
                     .style('align-items', 'center')
                     .style('justify-content', 'center')
-                    .append('div')
-                    .attr('class', 'y-label')
+                    .style('width', '1em')
+                    .select('div.y-label')
```

Comment:
So if we add `transform: rotate(-90deg)` to the css as the default (like Colin suggested), we should be able to avoid any additional styling of .y-label here.

Reply:
I'll have a look at that too.

Reply:
The default for `chartCartesian` is now to have the labels vertical (without clipping). It also seems to be very easy to make the label horizontal again using just CSS. I've tested this with axes on both sides of the chart.